.deploy/helpers/rollback.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-console */
+import childProcess from 'child_process';
+import process from 'process';
+
+import config from '../config.js';
+
+/**
+ * Rollback to previous deployed versions.
+ * @param {array} deployedServices - List of deployed services to rollback.
+ * @return {void}
+ */
+export default deployedServices => {
+  const regex = /Serverless: Timestamp: (\d+)/gs;
+  if (deployedServices.length === 0) {
+    console.log('Nothing to rollback!');
+  }
+
+  for (const deployedService of deployedServices) {
+    let output = '';
+    process.chdir(deployedService);
+
+    // List previous deployed versions of this service with deploy list command.
+    console.log(`Started rollback in: ${deployedService}`);
+    try {
+      output = childProcess.execSync(`${config.listCommand}`).toString();
+    } catch (_error) {
+      console.log('Error listing deployed releases!');
```

Comment:
Not sure about this, perhaps error messages are already logged as a result of the execSync? Otherwise maybe log some information about the error.